apps/prairielearn/src/tests/instructorQuestions.test.ts

```diff
@@ -59,30 +76,14 @@ describe('Instructor questions', function () {
       }
       questions = result.rows;
     });
-    it('should contain the addNumbers question', function () {
-      const question = _.find(questions, { directory: addNumbers.qid });
-      assert.isDefined(question);
-      addNumbers.id = question.id;
-    });
-    it('should contain the addVectors question', function () {
-      const question = _.find(questions, { directory: addVectors.qid });
-      assert.isDefined(question);
-      addVectors.id = question.id;
-    });
-    it('should contain the downloadFile question', function () {
-      const question = _.find(questions, {
-        directory: downloadFile.qid,
-      });
-      assert.isDefined(question);
-      downloadFile.id = question.id;
-    });
-    it('should contain the differentiatePolynomial question', function () {
-      const question = _.find(questions, {
-        directory: differentiatePolynomial.qid,
+
+    for (const testQuestion of testQuestions) {
+      it(`should contain the ${testQuestion.qid} question`, function () {
+        const foundQuestion = _.find(questions, { directory: testQuestion.qid });
+        assert.isDefined(foundQuestion);
+        testQuestion.id = foundQuestion.id;
```

Comment:
somewhat unrelated refactoring (though it is sort of related--if I hadn't added the for loop to de-dupe the code, I would have had to copy paste that same block yet another time for the `customElement` question I added tests for).